angular-workspace/projects/ni/nimble-angular/rich-text-editor/nimble-rich-text-editor.directive.ts

```diff
@@ -0,0 +1,93 @@
+import { Directive, ElementRef, EventEmitter, HostListener, Input, Output, Renderer2 } from '@angular/core';
+import type { RichTextEditor } from '@ni/nimble-components/dist/esm/rich-text-editor';
+import { BooleanValueOrAttribute, toBooleanProperty } from '@ni/nimble-angular/internal-utilities';
+
+export type { RichTextEditor };
+
+/**
+ * Directive to provide Angular integration for the rich text editor element.
+ */
+@Directive({
+    selector: 'nimble-rich-text-editor'
+})
+
+export class NimbleRichTextEditorDirective {
+    @Output() public inputEvent = new EventEmitter<boolean>();
+
+    public get disabled(): boolean {
+        return this.elementRef.nativeElement.disabled;
+    }
+
+    @Input() public set disabled(value: BooleanValueOrAttribute) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'disabled', toBooleanProperty(value));
+    }
+
+    public get footerHidden(): boolean {
+        return this.elementRef.nativeElement.footerHidden;
+    }
+
+    // Renaming because property should have camel casing, but attribute should not
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('footer-hidden') public set footerHidden(value: BooleanValueOrAttribute) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'footerHidden', toBooleanProperty(value));
+    }
+
+    public get fitToContent(): boolean {
+        return this.elementRef.nativeElement.fitToContent;
+    }
+
+    // Renaming because property should have camel casing, but attribute should not
+    // eslint-disable-next-line @angular-eslint/no-input-rename
+    @Input('fit-to-content') public set fitToContent(value: BooleanValueOrAttribute) {
+        this.renderer.setProperty(this.elementRef.nativeElement, 'fitToContent', toBooleanProperty(value));
+    }
+
+    public get errorVisible(): boolean {
+        return this.elementRef.nativeElement.errorVisible;
+    }
+
+    // errorVisible property maps to the error-visible attribute
```

Comment:
nit: Comment lines can be similar pattern

Reply:
Updated